### ISSUE TITLE
Undefined is not an object(evaluating react3.default.proptypes.string), does not work with latest react-native@0.49.1

### ISSUE THREAD

**Issue by yogendrakarekar** [tlenclos/react-native-audio-streaming#118]

**Comment by nickdirienzo** [comment]
@yogendrakarekar if you install from 2a4a95784426e55cc16e7acfd38c683b7d87f233, it'll work.

@tlenclos do you plan on releasing a version bump to publish the fix in npm?

**Comment by yogendrakarekar** [comment]
Hi,

Well, are you asking me whether I wish make corrections and release a new
version of this plugin to suit up with react native@0.49.9, well I don't
have any such plans.

But would love to help you out, take up a task, that you suggest, and do it.

Thanks & Regards
Yogendra U. Karekar
Mobile:-9769856284
Email:-yogendra.karekar@gmail.com

**Comment by nickdirienzo** [comment]
@yogendrakarekar no I wasn't suggesting that. There's a fix already in `master`, so you can have your `react-native-audio-streaming` dependency point to the SHA (or `#master`). Then it should work for you.

I was wondering if the maintainer had plans to publish a new version on npm which includes that fix since it's been in the repository for a while now.

**Comment by Stormcore** [comment]
Please bump the npm version

**Comment by TheeOMM** [comment]
@nickdirienzo what's the right way to point it master. I am currently just doing _npm i -S https://github.com/tlenclos/react-native-audio-streaming#master_. Is there another way to do it?

**Comment by BuffMcBigHuge** [comment]
@TheeOMM in your package.json, try `"react-native-autogrow-textinput": "git+https://github.com/wix/react-native-autogrow-textinput.git"`

**Comment by nickdirienzo** [comment]
@TheeOMM yes, what @BuffMcBigHuge said but replace `react-native-autogrow-textinput` with `react-native-audio-streaming`. Or you can do `npm install tlenclos/react-native-audio-streaming`. Hope this helps!

**Comment by BuffMcBigHuge** [comment]
@nickdirienzo Ah! Thanks for pointing that out! I was the autogrow package had the same issue!

**Comment by salisuabubakar** [comment]
Please ... am confuse how to install from #master. Can someone do step by step instruction. Thanks for your time.

**Comment by salisuabubakar** [comment]
Finally i was able to install but app crashes before launch. I think there is an issue using it with Wix React Native Navigation. Can someone help. Thanks

**Comment by Erenner** [comment]
import PropTypes from 'prop-types';
Player.propTypes = {
    url: PropTypes.string.isRequired
};
